Comp2024/src/main/java/frc/robot/subsystems/HID.java

```diff
@@ -51,7 +57,15 @@ public HID(GenericHID driver, GenericHID operator)
   @Override
   public void periodic( )
   {
-    // This method will be called once per scheduler run

```

Comment:
Please leave this comment in for the next person reading the code (that has less experience than you).